engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-subscriptions-list.html

```diff
@@ -36,8 +36,12 @@
 
       <tbody>
         <tr bst-table-row ng-repeat-start="(name, subscriptions) in groupedSubscriptions | groupedFilter: subscriptionSearch">
-          <td bst-table-cell class="row-select" style="white-space:nowrap">{{ name }}</td>
-          <td bst-table-cell colspan="8" ></td>
+          <td class="row-select"></td>
+          <td colspan="8">
+            <a href='/subscriptions/{{ subscription.id }}/info' class="confined-text">
```

Comment:
this doesn't work for me, as there is no 'subscription' variable here, only subscriptions. 'subscription' is defined below in the ng-repeat, but not here.